<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DirectCamView</class>
 <widget class="QWidget" name="DirectCamView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1331</width>
    <height>787</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_10">
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout_33" rowstretch="4,0">
     <item row="1" column="0">
      <layout class="QGridLayout" name="gridLayout_30">
       <item row="0" column="1">
        <layout class="QGridLayout" name="gridLayout_8">
         <item row="2" column="1">
          <layout class="QGridLayout" name="gridLayout_26">
           <item row="0" column="0">
            <widget class="QPushButton" name="pushButton_down">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>50</width>
               <height>50</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>50</width>
               <height>50</height>
              </size>
             </property>
             <property name="autoFillBackground">
              <bool>false</bool>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="icon">
              <iconset resource="../../build/client/resources.qrc">
               <normaloff>:/resources/down.png</normaloff>:/resources/down.png</iconset>
             </property>
             <property name="iconSize">
              <size>
               <width>36</width>
               <height>36</height>
              </size>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="1" column="0">
          <layout class="QGridLayout" name="gridLayout_21">
           <item row="0" column="0">
            <widget class="QPushButton" name="pushButton_left">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>50</width>
               <height>50</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>50</width>
               <height>50</height>
              </size>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="icon">
              <iconset resource="../../build/client/resources.qrc">
               <normaloff>:/resources/left.png</normaloff>:/resources/left.png</iconset>
             </property>
             <property name="iconSize">
              <size>
               <width>36</width>
               <height>36</height>
              </size>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="1" column="1">
          <layout class="QGridLayout" name="gridLayout_25"/>
         </item>
         <item row="2" column="0">
          <layout class="QGridLayout" name="gridLayout_20"/>
         </item>
         <item row="0" column="1">
          <layout class="QGridLayout" name="gridLayout_24">
           <item row="0" column="0">
            <widget class="QPushButton" name="pushButton_up">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>50</width>
               <height>50</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>50</width>
               <height>50</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true"/>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="icon">
              <iconset resource="../../build/client/resources.qrc">
               <normaloff>:/resources/up.png</normaloff>:/resources/up.png</iconset>
             </property>
             <property name="iconSize">
              <size>
               <width>36</width>
               <height>36</height>
              </size>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="0" column="0">
          <layout class="QGridLayout" name="gridLayout_23"/>
         </item>
         <item row="0" column="2">
          <layout class="QGridLayout" name="gridLayout_27"/>
         </item>
         <item row="1" column="2">
          <layout class="QGridLayout" name="gridLayout_28">
           <item row="0" column="0">
            <widget class="QPushButton" name="pushButton_right">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>50</width>
               <height>50</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>50</width>
               <height>50</height>
              </size>
             </property>
             <property name="baseSize">
              <size>
               <width>0</width>
               <height>0</height>
              </size>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="icon">
              <iconset resource="../../build/client/resources.qrc">
               <normaloff>:/resources/right.png</normaloff>:/resources/right.png</iconset>
             </property>
             <property name="iconSize">
              <size>
               <width>36</width>
               <height>36</height>
              </size>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="2" column="2">
          <layout class="QGridLayout" name="gridLayout_29"/>
         </item>
        </layout>
       </item>
       <item row="0" column="0">
        <spacer name="horizontalSpacer_4">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="0" column="2">
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item row="0" column="0">
      <widget class="QWidget" name="widget" native="true">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(238, 238, 236);</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_9">
        <item row="0" column="0">
         <layout class="QGridLayout" name="gridLayout_6" rowstretch="0,0,0,0,0,0,0" columnstretch="0,0,0,0,0,0,0">
          <item row="2" column="5">
           <spacer name="verticalSpacer_2">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="0" column="3">
           <spacer name="verticalSpacer_4">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeType">
             <enum>QSizePolicy::Fixed</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="2" column="2">
           <spacer name="horizontalSpacer_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="4" column="5">
           <spacer name="verticalSpacer_3">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="6" column="3">
           <spacer name="verticalSpacer_5">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeType">
             <enum>QSizePolicy::Fixed</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="3" column="0">
           <spacer name="horizontalSpacer_5">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeType">
             <enum>QSizePolicy::Fixed</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="3" column="1">
           <layout class="QGridLayout" name="gridLayout_12">
            <item row="0" column="0" alignment="Qt::AlignHCenter">
             <widget class="QRadioButton" name="radioButton_ml">
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="5" column="5">
           <layout class="QGridLayout" name="gridLayout_19">
            <item row="0" column="0" alignment="Qt::AlignHCenter">
             <widget class="QRadioButton" name="radioButton_br">
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="2" column="3">
           <layout class="QGridLayout" name="gridLayout_35"/>
          </item>
          <item row="4" column="3">
           <layout class="QGridLayout" name="gridLayout_36"/>
          </item>
          <item row="5" column="1">
           <layout class="QGridLayout" name="gridLayout_11">
            <item row="0" column="0" alignment="Qt::AlignHCenter">
             <widget class="QRadioButton" name="radioButton_bl">
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="1" column="3">
           <layout class="QGridLayout" name="gridLayout_14">
            <item row="0" column="0" alignment="Qt::AlignHCenter">
             <widget class="QRadioButton" name="radioButton_tm">
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="3" column="3">
           <layout class="QGridLayout" name="gridLayout_16">
            <item row="0" column="0" alignment="Qt::AlignHCenter">
             <widget class="QRadioButton" name="radioButton_mm">
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="3" column="4">
           <layout class="QGridLayout" name="gridLayout_37"/>
          </item>
          <item row="3" column="5">
           <layout class="QGridLayout" name="gridLayout_17">
            <item row="0" column="0" alignment="Qt::AlignHCenter">
             <widget class="QRadioButton" name="radioButton_mr">
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="1" column="1">
           <layout class="QGridLayout" name="gridLayout_13">
            <item row="0" column="0" alignment="Qt::AlignHCenter">
             <widget class="QRadioButton" name="radioButton_tl">
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="3" column="2">
           <layout class="QGridLayout" name="gridLayout_31"/>
          </item>
          <item row="5" column="3">
           <layout class="QGridLayout" name="gridLayout_18">
            <item row="0" column="0" alignment="Qt::AlignHCenter">
             <widget class="QRadioButton" name="radioButton_bm">
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="1" column="5">
           <layout class="QGridLayout" name="gridLayout_15">
            <item row="0" column="0" alignment="Qt::AlignHCenter">
             <widget class="QRadioButton" name="radioButton_tr">
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="2" column="4">
           <spacer name="horizontalSpacer_3">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="3" column="6">
           <spacer name="horizontalSpacer_6">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeType">
             <enum>QSizePolicy::Fixed</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="1">
    <layout class="QGridLayout" name="gridLayout_7">
     <item row="0" column="0">
      <layout class="QGridLayout" name="gridLayout_5">
       <item row="0" column="0">
        <widget class="QGroupBox" name="groupBox">
         <property name="title">
          <string>可见光直显</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_2">
          <item row="0" column="0">
           <layout class="QGridLayout" name="gridLayout">
            <item row="5" column="0">
             <widget class="QPushButton" name="pushButton_a6">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="text">
               <string>Cam6</string>
              </property>
             </widget>
            </item>
            <item row="7" column="0">
             <widget class="QPushButton" name="pushButton_a8">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="text">
               <string>Cam8</string>
              </property>
             </widget>
            </item>
            <item row="6" column="0">
             <widget class="QPushButton" name="pushButton_a7">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="text">
               <string>Cam7</string>
              </property>
             </widget>
            </item>
            <item row="4" column="0">
             <widget class="QPushButton" name="pushButton_a5">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="text">
               <string>Cam5</string>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QPushButton" name="pushButton_a2">
              <property name="enabled">
               <bool>true</bool>
              </property>
              <property name="text">
               <string>Cam2</string>
              </property>
              <property name="checkable">
               <bool>false</bool>
              </property>
             </widget>
            </item>
            <item row="11" column="0">
             <widget class="QPushButton" name="pushButton_a12">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="text">
               <string>Cam12</string>
              </property>
             </widget>
            </item>
            <item row="2" column="0">
             <widget class="QPushButton" name="pushButton_a3">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="text">
               <string>Cam3</string>
              </property>
             </widget>
            </item>
            <item row="10" column="0">
             <widget class="QPushButton" name="pushButton_a11">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="text">
               <string>Cam11</string>
              </property>
             </widget>
            </item>
            <item row="0" column="0">
             <widget class="QPushButton" name="pushButton_a1">
              <property name="enabled">
               <bool>true</bool>
              </property>
              <property name="text">
               <string>Cam1</string>
              </property>
             </widget>
            </item>
            <item row="8" column="0">
             <widget class="QPushButton" name="pushButton_a9">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="text">
               <string>Cam9</string>
              </property>
             </widget>
            </item>
            <item row="9" column="0">
             <widget class="QPushButton" name="pushButton_a10">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="text">
               <string>Cam10</string>
              </property>
             </widget>
            </item>
            <item row="3" column="0">
             <widget class="QPushButton" name="pushButton_a4">
              <property name="text">
               <string>Cam4</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QGroupBox" name="groupBox_2">
         <property name="title">
          <string>红外光直显</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_3">
          <item row="0" column="0">
           <layout class="QGridLayout" name="gridLayout_4">
            <item row="8" column="0">
             <widget class="QPushButton" name="pushButton_b9">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="text">
               <string>Cam9</string>
              </property>
             </widget>
            </item>
            <item row="4" column="0">
             <widget class="QPushButton" name="pushButton_b5">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="text">
               <string>Cam5</string>
              </property>
             </widget>
            </item>
            <item row="11" column="0">
             <widget class="QPushButton" name="pushButton_b12">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="text">
               <string>Cam12</string>
              </property>
             </widget>
            </item>
            <item row="6" column="0">
             <widget class="QPushButton" name="pushButton_b7">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="text">
               <string>Cam7</string>
              </property>
             </widget>
            </item>
            <item row="0" column="0">
             <widget class="QPushButton" name="pushButton_b1">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="text">
               <string>Cam1</string>
              </property>
             </widget>
            </item>
            <item row="9" column="0">
             <widget class="QPushButton" name="pushButton_b10">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="text">
               <string>Cam10</string>
              </property>
             </widget>
            </item>
            <item row="5" column="0">
             <widget class="QPushButton" name="pushButton_b6">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="text">
               <string>Cam6</string>
              </property>
             </widget>
            </item>
            <item row="7" column="0">
             <widget class="QPushButton" name="pushButton_b8">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="text">
               <string>Cam8</string>
              </property>
             </widget>
            </item>
            <item row="10" column="0">
             <widget class="QPushButton" name="pushButton_b11">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="text">
               <string>Cam11</string>
              </property>
             </widget>
            </item>
            <item row="2" column="0">
             <widget class="QPushButton" name="pushButton_b3">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="text">
               <string>Cam3</string>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QPushButton" name="pushButton_b2">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="text">
               <string>Cam2</string>
              </property>
             </widget>
            </item>
            <item row="3" column="0">
             <widget class="QPushButton" name="pushButton_b4">
              <property name="text">
               <string>Cam4</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </item>
     <item row="1" column="0">
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../build/client/resources.qrc"/>
 </resources>
 <connections/>
</ui>
